---
title: 'PGS-220000 Error de PostgreSQL: la relación "t1" no existe'
description: Obtenga información sobre cómo corregir el error generado por el flujo de trabajo de seguimiento después de una actualización de compilación reciente 9343 o superior en Adobe Campaign Classic v7.
solution: Campaign Classic v7
product: Campaign Classic v7
applies-to: Campaign Classic v7
keywords: KCS, Campaign Classic, flujo de trabajo de seguimiento, nms:TrackingLog
resolution: Resolution
internal-notes: null
bug: false
article-created-by: Amol Mahajan
article-created-date: 11/16/2023 10:39:55 AM
article-published-by: Amol Mahajan
article-published-date: 11/16/2023 10:51:09 AM
version-number: 4
article-number: KA-17530
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=65d80679-6c84-ee11-8179-6045bd0065b6
exl-id: 1a8a9bfd-7dcf-4a7d-b87e-0f17e6e63a08
source-git-commit: 362aef9e63f8a0303b670074f58e19d75587bfeb
workflow-type: tm+mt
source-wordcount: '198'
ht-degree: 2%

---

# Error de PGS-220000 PostgreSQL: la relación &quot;t1&quot; no existe


Si aparece un error: `PGS-220000 PostgreSQL error: relation "t1" does not exist...,` generado por el flujo de trabajo de seguimiento en Adobe Campaign Classic v7, consulte este artículo para saber cómo resolver este problema.

## Descripción {#description}


### <b>Entorno</b>

Campaign Classic V7



### <b>Problema/Síntomas</b>

Después de una actualización de compilación 9343 o superior reciente, el flujo de trabajo de seguimiento genera los siguientes errores:




```
nlmodule    PGS-220000 PostgreSQL error: ERROR:  relation "t1" does not exist LINE 1: ...t, T3.iArticle, T2.iType FROM NmsTrackingUrl T2,  T1 LEFT OU... nlmodule    WDB-200001 SQL statement 'SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2,  T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog < :#(1)# AND T1.tsLog >= :#(2)#  AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog' could not be executed.   Param(0)=09/17/2020 8:55:00 PM   Param(1)=09/16/2020 8:55:00 PMnlmodule    Attribute 'broadLog-id' unknown (see definition of schema 'Tracking logs (nms:trackingLog)').
```





## Resolución {#resolution}


Desde los registros de seguimiento detallados, se puede ver que el flujo de trabajo de seguimiento falla al final, al consolidar los datos:




```
00000000 00000000 1 info log Consolidating tracking logs for the period [ 09/16/2020 3:55:00 PM, 09/17/2020 3:55:00 PM[ ...00000000 00000000 1 info wdbc Select: [ 00000000 ]  SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog < :#(1)# AND T1.tsLog >= :#(2)# AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog00000000 00000000 2 info wdbc Param(0)=09/17/2020 8:55:00 PM00000000 00000000 2 info wdbc Param(1)=09/16/2020 8:55:00 PM00000000 00000000 1 error log PGS-220000 PostgreSQL error: ERROR: relation "t1" does not exist\nLINE 1: ...t, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OU...\n \n. (iRc=-2006)00000000 00000000 1 error log WDB-200001 SQL statement 'SELECT T1.tsLog, T1.iDeliveryId, T1.iUrlId, T1.sSourceType, T3.dAmount, T3.iArticle, T2.iType FROM NmsTrackingUrl T2, T1 LEFT OUTER JOIN NmsWebTrackingLog T3 ON iTrackingLogId = iWebTrackingLogId WHERE T1.tsLog < :#(1)# AND T1.tsLog >= :#(2)# AND T1.iUrlId=T2.iTrackingUrlId ORDER BY T1.tsLog' could not be executed.\n Param(0)=09/17/2020 8:55:00 PM\n Param(1)=09/16/2020 8:55:00 PM (iRc=-2006)
```




Cada vez que se inicia el flujo de trabajo de seguimiento, extrae todas las asignaciones de destino y actualiza los datos en función de la configuración.

En este caso, el problema se debe a una asignación de destino. Consulte lo siguiente:

![](assets/a06a8deb-6536-ec11-b6e6-000d3a348885.png)

Básicamente,<b> NMS: registro de seguimiento</b> Este esquema NO debe utilizarse para registros de monitorización, ya que no está relacionado con una tabla de base de datos (SQL).

En conclusión, el problema no tiene nada que ver con la actualización, sino más con una configuración de asignación de destino incorrecta.
